src/transformers/models/albert/modeling_tf_albert.py

```diff
@@ -246,6 +251,7 @@ def __init__(self, config: AlbertConfig, **kwargs):
         # Two different dropout probabilities; see https://github.com/google-research/albert/blob/master/modeling.py#L971-L993
         self.attention_dropout = tf.keras.layers.Dropout(rate=config.attention_probs_dropout_prob)
         self.output_dropout = tf.keras.layers.Dropout(rate=config.hidden_dropout_prob)
+        self.config = config
```

Comment:
Storing the whole config in each layer can start to make things a lot bigger if it's large e.g. `id2label` with many classes. In the case of TF with safetensors, are we just instantiating the class from the library and loading the weight e.g. we don't store all the class attributes when saving?

Reply:
This is a good point, but I think it's okay! Our `save_pretrained` methods should just save the weights, and not save attributes of the layers like this. Also, setting `self.config = config` just creates a reference to the same underlying `config` object, so it doesn't use extra memory when the model is initialized either.

Reply:
In general, a lot of our models do `self.config = config` in the `__init__` anyway! I just needed to add a lot more of it so that the `build()` methods could see the config vars they need.